chef/cookbooks/rabbitmq/files/suse/epmd.socket-listen.conf

```diff
@@ -0,0 +1,3 @@
+[Socket]
+ListenStream=
```

Comment:
Yes; there can be multiple ListenStream values and without this, we append our value instead of replacing the old values with the new one.